extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -20,7 +20,18 @@ import {user} from '../../../src/log';
 
 const TAG = 'amp-auto-ads';
 const STICKY_AD_TAG = 'amp-sticky-ad';
-const OPT_IN_STATUS_ANCHOR_ADS = 2;
+
+/**
+ * Indicates Opt in status of the publisher for different ad formats.
+ * @enum {number}
+ */
+const OptInStatus = {
+  UNKNOWN: 0,
+  OPT_IN_STATUS_BANNER_ADS: 1,
```

Comment:
This doesn't seem like the right location for this enum, since a number of the values are not relevant to anchor-ad-strategy. If you want to add an enum like this, then please move it to ad-network-config.js or its own separate file.

Reply:
Moved to separate file.